scripts/secureli-deployment.sh

```diff
@@ -3,6 +3,7 @@ set -exo pipefail
 
 pip install requests jinja2 poetry python-semantic-release # TODO: Look into Dockerizing these tools so we're not pulling them down each pipeile run
 export secureliVersion=$(semantic-release print-version --current)
+cd homebrew-secureli
```

Comment:
Adding the CD before the conditional. Technically semver pushes the new tag before the deploy pipeline is triggered so my original logic was flawed since I was checking the tag in the secureli repo & not homebrew-secureli